src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -9,6 +9,7 @@ import filter from 'lodash/filter';
 import memoize from 'lodash/memoize';
 import capitalize from 'lodash/capitalize';
 import useDeepCompareEffect from 'use-deep-compare-effect';
+import uniqBy from 'lodash/uniqBy';
```

Comment:
FYI, I believe this can by done by Sets now 👍 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
again, force of habit, thanks!